src/serial.rs

```diff
@@ -93,14 +91,14 @@ pub enum StopBits {
 }
 
 pub struct Config {
-    pub baudrate: Bps,
+    pub baudrate: BitsPerSecond,
```

Comment:
This isn't right, is it? I think we need `BytesPerSecond`.

(I don't think it makes a difference in functionality, but it's the wrong type semantically. Could lead to confusion.)

Reply:
Turns out there's a `Baud` struct which is far better suited lol. I force pushed to the branch with a fix for `BitsPerSecond` to `Baud`.